spec/factories/interrupt_page_acknowledgements.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :interrupt_page_acknowledgement do
+    user
+    recruitment_cycle
+    page { :rollover }
```

Comment:
Can we do traits for the other one please.

Reply:
Seems weird to make a trait that is just altering a single attribute. That aren't going to be used anywhere?